src/Microsoft.ML.FastTree/FastTreeRegression.cs

```diff
@@ -494,7 +494,7 @@ private static FastTreeRegressionModelParameters Create(IHostEnvironment env, Mo
         public override PredictionKind PredictionKind => PredictionKind.Regression;
     }
 
-    public static partial class FastTree
+    public static partial class FastTreeEntryPoint
```

Comment:
>FastTreeEntryPoint [](start = 32, length = 18)

FYI I have a change to make this internal, so perhaps this will not be necessary.

Reply:
Ok. I will handle some merge conflicts but let me keep this for now for building it on CI.

---
In reply to: [251652346](https://github.com/dotnet/machinelearning/pull/2243#discussion_r251652346) [](ancestors = 251652346)

Reply:
It'd be nice to make it general. Having the same name of class and namespace doesn't look very good to me.

---
In reply to: [251949357](https://github.com/dotnet/machinelearning/pull/2243#discussion_r251949357) [](ancestors = 251949357)